featurebyte/common/join_utils.py

```diff
@@ -56,7 +85,9 @@ def filter_columns(columns: List[str], exclude_columns: List[str]) -> List[str]:
     return [col for col in columns if col not in exclude_columns_set]
 
 
-def append_rsuffix_to_columns(columns: List[str], rsuffix: Optional[str]) -> List[str]:
+def apply_column_name_modifiers(
+    columns: List[str], rsuffix: Optional[str], rprefix: Optional[str]
```

Comment:
do we want a default value for `rprefix` since it seems like most of the current calls are `None`?

Reply:
I made it mandatory to make sure I didn't miss any places that should be updated to pass in this parameter. I think that should be fine since this is for internal use only. For the top level user facing method, there is a default value.